ui/app/components/mfa/mfa-login-enforcement-form.js

```diff
@@ -49,6 +49,7 @@ export default class MfaLoginEnforcementForm extends Component {
     selected: [],
   };
   @tracked authMethods = [];
+  @tracked mfaMethods = [];
```

Comment:
Does this update while the user manipulates the dom? If not, then it doesn't have to be tracked since it's just set once in the constructor.

Reply:
Good call! I'll amend.